src/components/data_submitter/ConsentGroupForm.js

```diff
@@ -0,0 +1,590 @@
+import { useState, useEffect, useCallback } from 'react';
+import { div, input, label, h, span, a, button } from 'react-hyperscript-helpers';
+import { cloneDeep, isNil } from 'lodash/fp';
+import {RadioButton} from '../RadioButton';
+import DataSubmitterStyles from './DataSubmitterStyles';
+import ConsentGroupSummary from './ConsentGroupSummary';
+import Select from 'react-select';
+
+// show text iff text
+const ConditionalText = (props) => {
+  const {
+    text,
+    setText,
+    id,
+    required,
+    placeholder
+  } = props;
+
+  const show = useCallback(() => {
+    return !isNil(text);
+  }, [text]);
+
+  return div({
+    isRendered: show(),
+  }, [
+    input({
+      type: 'text',
+      style: DataSubmitterStyles.textInput,
+      value: text,
+      onChange: (e) => setText(e.target.value),
+      name: id,
+      className: 'form-control',
+      id: id,
+      maxLength: '512',
+      rows: '2',
+      required: required,
+      placeholder: placeholder
+    })
+  ]);
+};
+
+
+const dataLocationOptions = [
+  { value: 'AnVIL Workspace', label: 'AnVIL Workspace' },
+  { value: 'Terra Workspace', label: 'Terra Workspace' },
+  { value: 'TDR Location', label: 'TDR Location' },
+  { value: 'Not Determined', label: 'Not Determined' }
+];
+
+// const dataLocationOptions = ["AnVIL Workspace", "Terra Workspace", "TDR Location", "Not Determined"]
+
+export const ConsentGroupForm = (props) => {
+  const {
+    idx,
+    parentConsentGroup,
+    saveConsentGroup,
+    deleteConsentGroup,
+  } = props;
+
+  const [consentGroup, setConsentGroup] = useState({
+    ...{
```

Comment:
can we move this variable out with a name to describe it? like `defaultConsentGroupValues`?